qdrant_client/qdrant_client.py

```diff
@@ -73,6 +73,19 @@ def __init__(
         path: Optional[str] = None,
         **kwargs: Any,
     ):
+        # Check if fastvector is installed
+        try:
+            from fastvector.qdrant_mixin import QdrantClientMixin
```

Comment:
If you import `fastvector`, then `fastvectors` should not depend on qdrant-client here https://github.com/NirantK/fastvector/blob/main/pyproject.toml#L15

Otherwise it is circular dependency

Reply:
Great catch! I've removed the `qdrant-client` dependency from `fastvectors`